docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+
+To flash ELRS receivers, you can use esptool or a Chrome or Chromium based browser as instructed [here](ELRS_RECEIVERS.md#flashing)
+
+[Flashing firmware](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-3) to the MatekSys mLRS Tx modules is especially easy.
+
+Once the firmware is installed, you can confirm operation by observing the LED.
+- When either a receiver or Tx module is powered on by itself, a LED (usually red) should flash at 2 Hz (twice per second, on for 1/4 second then off for 1/4 second repeatedly).
+- When the Tx module and receiver are both powered on and bound, a LED (usually green) should flash more slowly at 1 Hz (once per second, on for 1/2 second then off for 1/2 second repeatedly) to indicate they are connected and exchanging radio messages.
+
+## 5. Install the mLRS Lua script and configure your mLRS system
+
+Follow the [instructions](LUA.md) and copy the correct mLRS Lua script to the SCRIPTS\TOOLS folder on your SD card and setup your RC radio external module.
+
+Start mLRS Configurator Lua script from the SYS->TOOLS menu and confirm you can talk to the Tx module.  You should see the TX module firmware version reported at the top of the screen.
+
+Use the mLRS Lua script to Confirm the receiver is connected and the firmware versions are recent enough to support the features you need.
+- If the radio link does not connect, follow the [Binding](BINDING.md) procedure.
+- It is HIGHLY recommended to use the same firmware version on both the receiver and Tx module. The latest released firmware is also recommended, especially for new installs.  If your Rx and Tx module firmware versions don't match or are very much out of date, return to the previous step and install the latest mLRS firmware release.
+
+Use the mLRS Lua script to configure your mLRS system
+- Leave your Rx powered on and connected while you complete the following steps.
+- Set the correct RF Band for your region (If using 900 MHz hardware).
+- You can also change the Bind Phrase if desired.
+- Set your desired Mode
+- If you didn't connect sbus or CRSF RC output from your receiver to your flight controller, enter the "Edit Rx" page and set Snd RcChannels to "rc override" (or "rc channels" for ArduPilot 4.6 with RC Channels support).  Return to the main page.  If your radio has a black and white screen, you will need to use the [CLI](CLI.md) for this.
+- Select "Save" and verify that your Rx reconnects.
+- You may wish to review and adjust the Rx and Tx power settings as well as other settings at this time, but the defaults should be OK to get started.
+
+## 6. Optionally, Install the Yaapu Telemetry script
+
+You may wish to install the [Yaapu Telemetry script](https://github.com/yaapu/FrskyTelemetryScript) which uses the CRSF telemetry messages which mLRS generates on your Tx module from the MAVLink data it receives when the Rx Serial Link Mode is set to "MAVLink" or "MAVLinkX".
```

Comment:
isn't just "... wish .." instead of ".. may wish ..." ok

since option names please "... is set to "mavlinkX" or "mavlink"."
fyi, "mavlinX" is recommended these days

Reply:
> isn't just "... wish .." instead of ".. may wish ..." ok

I don't think so.  In this sentence,  "You may wish" means "you can decide" and we don't know what you want.
"You wish" would mean "you have decided" and we know you want to do it.